mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -77,6 +77,20 @@ class Word2Vec extends Serializable with Logging {
   private var numIterations = 1
   private var seed = Utils.random.nextLong()
   private var minCount = 5
+  private var maxSentenceLength = 1000
+
+  /**
+   * sets the maxSentenceLength, maxSentenceLength is used as the threshold for cutting sentence
```

Comment:
Sorry to be pedantic, but can we capitalize the `sets` and make the comment something like:

```
Sets the maximum length of each sentence in the input data. Any sentence longer than this threshold will be divided into chunks of up to `maxSentenceLength` size (default: 1000)
```